### ISSUE TITLE
directory for the config file

### ISSUE THREAD

**Issue by sichendart** [tomography/XRFtomo#56]
By doing the following, the xrftomo.config was generated under /Users/sichen

(XRFtomo) Ahri-59:XRFtomo sichen$ pwd
/Users/sichen/src/XRFtomo
(XRFtomo) Ahri-59:XRFtomo sichen$ XRFtomo init

It makes more sense to generate it under /Users/sichen/src/XRFtomo. I also check the directory for the configure file when the software is quit. It is under /Users/sichen/src/XRFtomo.

**Comment by decarlof** [comment]
Actually this change was intentional: Most of the users will not built the code from source by rather install it from a conda channel. In this case is better to save the config in a location that always exists (user home) rather than the directory from where it was lunched.

**Comment by sichendart** [comment]
OK. In that case, shall we always save the config file in the user home directory, for example, when we save from the menu or when the software quits.

**Comment by decarlof** [comment]
correct, the config file is designed to keep the status of the software on file exit, its location is set in this [line](https://github.com/tomography/XRFtomo/blob/cefc9f441688b26e0dc58849f13fd707f72504e9/xrftomo/config.py#L12) of the [config.py](https://github.com/tomography/XRFtomo/blob/master/xrftomo/config.py) file and stays the same so the program can restore its status at startup as well save it on quit.

**Comment by FabricioSMarin** [comment]
I find that xrftomo.conf gets created in the bin directory sometimes.

**Comment by decarlof** [comment]
This in not the intentional behavior: there should be only one config file.  In this  [line](https://github.com/tomography/XRFtomo/blob/cefc9f441688b26e0dc58849f13fd707f72504e9/xrftomo/gui.py#L1105) the config file name is hard coded. This should be changed in:
```
config.write(args.config, args=self.params, sections=sections)
```

**Comment by FabricioSMarin** [comment]
I'm using `expander("~")` to find the home path and write the config file there.  Would something like this work? So far I haven't encountered issues with starting XRFtomo from any window or path. 

```
from os.path import expanduser            

home = expanduser("~")
config.write('{}/xrftomo.conf'.format(home), args=self.params, sections=sections)
```
